plugin/httptrace/httptrace.go

```diff
@@ -41,7 +42,7 @@ var (
 	encoding = binary.BigEndian
 )
 
-// Returns the Attributes, Context Tags, and SpanContext that were encoded by Inject.
+// Extract returns the Attributes, Context Tags, and SpanContext that were encoded by Inject
```

Comment:
nit: drop "returns", that is documented below.

Reply:
Seems like this might be the absolute nittiest of nits ;) Exportable method comments have a proper format of `// MyMethodName does something awesome`, so generally the second word is a verb, which is often "returns" (e.g., in `net/http` builtin package, the export comment for CanonicalHeaderKey). Happy to use another verb if you have any recommendations, but I am fresh out of ideas. Will think on it some and revisit in just a bit.

Reply:
I marked it as "nit", so feel free to ignore, but here is my reasoning:

What the function returns is already part of it's signature. I have no problem with *ending* with `.... It returns the Attributes, Context Tags & SpanContext that were encoded by Inject.` but ATM that is the *only* bit of info.

As I said, you can ignore this nit, but the documentation doesn't answer these questions:
Why are `Vendor` TraceState bits dropped?
Why does Attributes (which is a []core.KeyValue ?) contain only a single value?
Why is that value the URL of the request?
Why is no error returned when tracecontext.FromHeaders errors?

To me, as someone who is trying to understand the shape of this API, these are all *way* more important than detailing what the function returns.

Reply:
@freeformz your reasoning is good.

@jranson as @freeformz pointed it is a nit with some rework to be implemented so it is fine to go as it is. Let's move ahead with your PR. That said now I have a nit of my own (but low cost): you dropped the punctuation at the end of the sentence :) Could you please update/rebase so we can merge it?